test/io/sungeun/ioerror.EACCES.good

```diff
@@ -1,3 +1,3 @@
-uncaught SystemError: Permission denied EACCES with path "this/is/my/path" offset -1
+uncaught SystemError: Input/output error (Permission denied EACCES with path "this/is/my/path" offset -1)
```

Comment:
This one looks weird, but it's in fact just because the test does something weird in this case (it doesn't actually pass the EACCESS code).